src/beheer-header/beheer-header.scss

```diff
@@ -67,6 +71,106 @@ beheer-header {
       text-decoration: none;
     }
 
+    .menu-button {
+      height: $header-height;
+
+      &>div {
+        height: 100%;
+        display: flex;
+        align-items: center;
+        gap: 0.5rem;
+      }
+
+      .dropdown {
+        position: relative;
+        z-index: 20;
+
+        display: flex;
+        align-items: center;
+
+        margin: 0;
+        height: 100%;
+
+        font-size: 0.875rem;
+
+        b {
+          font-size: 0.875rem;
+        }
+
+        &:hover {
+          .dropdown__toggle {
+            background-color: $black;
+          }
+
+          .dropdown__content {
+            max-height: 500px;
+            overflow: auto;
+          }
+        }
+
+        .dropdown__toggle {
```

Comment:
Dit veranderen naar dropdown-toggle dus wss &-toggle { }

Reply:
Overal die classnames met dashes doen ipv underscores

Reply:
ik ga het zo laten om niks meer te breken nu, is op zich wel ok met BEM naming conventions van classes ze